scikeras/wrappers.py

```diff
@@ -250,6 +255,23 @@ def _fit_keras_model(self, X, y, sample_weight, warm_start, **kwargs):
         # order implies kwargs overwrites fit_args
         fit_args = {**fit_args, **kwargs}
 
+        if OS_IS_WINDOWS:
+            # see tensorflow/probability#886
+            if not isinstance(X, np.ndarray):  # list, tuple, etc.
```

Comment:
I might make a small helper function to remove some for-loops/if-staments:

``` python
def _windows_upcast_ints(x: np.ndarray) -> np.ndarray:
    return x.astype("int64") if x.dtype == np.int32 else x

if OS_IS_WINDOWS:
    X = _windows_upcast_int(X) if isinstance(X, np.ndarray) else [_windows_upcast_int(x) for x in X]
    y = _windows_upcast_int(y) if isinstance(y, np.ndarray) else [_windows_upcast_int(yi) for yi in y]
```

Reply:
Yeah that would help, please feel free to implement!